framework-test/src/main/java/org/checkerframework/framework/test/TypecheckExecutor.java

```diff
@@ -75,33 +75,11 @@ public CompilationResult compile(TestConfiguration configuration) {
             options.add("-processor");
             options.add(String.join(",", configuration.getProcessors()));
 
-            List<String> nonJvmOptions = new ArrayList<>();
             for (String option : configuration.getFlatOptions()) {
                 if (!option.startsWith("-J-")) {
-                    nonJvmOptions.add(option);
+                    options.add(option);
                 }
             }
-            nonJvmOptions.add("-Xmaxerrs");
-            nonJvmOptions.add("100000");
```

Comment:
This overrides the previous number -- `9999`. I'm removing it because `9999` should be large enough.